### PR TITLE
raidboss: Fix e9s callout not being declared properly

### LINKED ISSUE #2995: e9s some triggers are not working after recent update
## Description

Those triggers alerted after these triggers below are not working.
E9S The Second Art Of Darkness Right
E9S The Second Art Of Darkness Left
E9S The Third Art Of Darkness Right
E9S The Third Art Of Darkness Left

I guess these trigger `E9S The Second / Third Art Of Darkness Charge Solver` and `E9S The Second / Third Art Of Darkness Callout` are not working.

Don't know if this happens on KR server only. I heard other person also had this problem.

I've checked tuesday working well. This happened after the recent update.

### PR DESCRIPTION
**Pull request by valarnin**
Noticed this issue while testing the language fix for raidemulator.

Edit:

Fixes #2995.

### CODE REVIEW

**Review by panicstevenson — APPROVED** [review]

File: ui/raidboss/data/05-shb/raid/e9s.js [hunk 1]
```diff
@@ -538,7 +538,7 @@ export default {
       durationSeconds: (data) => data.finalArtOfDarkness ? 16 : 9,
       alertText: (data, _matches, output) => {
         // Perform the callout now, regardless if it's The Second or Third Art Of Darkness
-        callout = data.artOfDarkness.slice();
+        const callout = data.artOfDarkness.slice();
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e9s.js:541]: Can you explain how this fixes #2995?  It seems like even if `callout` is a global here, that it gets clobbered in this function and so I'm not sure I follow how the behavior changes by adding `const`.

Comment (valarnin) [reply ui/raidboss/data/05-shb/raid/e9s.js:541]: Except that it's not a global, it's a local, and it's being used without being declared.

image

**Comment by Trim21** [comment]
our lint should check this.

File: ui/raidboss/data/05-shb/raid/e9s.js (hunk 1 above)
Comment (quisquous) [reply ui/raidboss/data/05-shb/raid/e9s.js:541]: Oh, it's `use strict` that makes this an error.  Got it.

**Pull request merged**